e2e/cypress/integration/main.spec.js

```diff
@@ -0,0 +1,163 @@
+function login(email, password) {
```

Comment:
Can we move all these functions to separate file? It will be easier to import them and reuse them if needed in other tests :)

Reply:
Good idea